### ISSUE TITLE
Browsersync causing a full refresh instead of injecting CSS & images. 

### ISSUE THREAD

**Issue by heavymery** [mgechev/angular2-seed#804]
> tools/utils/seed/code_change_tools.ts
```js
let changed = (files: any) => {
  if (!(files instanceof Array)) {
    files = [files];
  }
  // if (ENABLE_HOT_LOADING) {
  //   ng2HotLoader.onChange(files);
  // } else {
  //TODO: Figure out why you can't pass a file to reload
  console.log(files);
  browserSync.reload(files);
  //}
};
```
https://github.com/mgechev/angular2-seed/commit/e3696aa15410a6befc0a37bb8732e4dc2693e924#commitcomment-16994199

@d3viant0ne 
Just as @gkalpak's [comment](https://github.com/mgechev/angular2-seed/commit/e3696aa15410a6befc0a37bb8732e4dc2693e924#commitcomment-16994199), `files.path` always be `undefined`. So, Browsersync causing a full refresh instead of injecting CSS & images. 

Because `files` will always be Array and `browserSync.reload` accept String or Array, We can call `browserSync.reload` with `files`.

>https://www.browsersync.io/docs/api/#api-reload
### Browsersync API
**.reload( arg )**
...
**arg**
Type: String | Array | Object [optional]

Then, injecting CSS & images will working correctly.

**Comment by ludohenin** [comment]
This is solved by bb00b19e5fab48e00a99aecda66f881dd59a2b35

**Comment by heavymery** [comment]
:metal:
